app/src/routes/examples/example7/MyComponent.svelte

```diff
@@ -1,24 +1,32 @@
 <script lang="ts">
-	import type { Instructs } from '$lib/components/PowerTable.svelte';
+	import { createEventDispatcher } from 'svelte';
+	import type { Cell } from '$lib/components/PowerTable.svelte';
+	import { EDIT_CONSTS } from '$lib/components/PowerTable.svelte'
 
-	export var ptInstructs: Instructs;
-	export var value: string;
-	export var rowIndex: number;
-	export var rowId: number;
+	export var cell: Cell;
 	export var selectValues: Array<string>;
+
+	const dispatch = createEventDispatcher();
+
+	function submitEdits(e: Event) {
+		dispatch(EDIT_CONSTS.SUBMISSION_EVENT, {
+			cell: cell,
+			domEvent: e
+		});
+	}
 </script>
 
 <label>
```

Comment:
My thinking here was that this example intentionally only implements the minimum markup needed for the functionality to work, not the styling. But I see why it would be good to have the consistency (and also get the CSS rules back).

Would it be worth doing two example components? The fixed version of above, and another component with some custom styling?

Reply:
I think an example of a non-editable component could be useful, perhaps for the `id` column.

Reply:
> I think an example of a non-editable component could be useful, perhaps for the `id` column.

I didn't think about non-editable component. Yes, that would be cool.

Reply:
Added in c294993